app/views/drivers/index.html.slim

```diff
@@ -0,0 +1,13 @@
+table class="table"
+  thread
+    tr
+      th scope="col" Name
+      th scope="col" License Number
+      th scope="col" Mobile Number
+      th scope="col" Email Address
+      th scope="col" Age
+      th scope="col" Notes
+      th scope="col" Status
+      th scope="col" Actions
+  tbody
+    = render partial: 'shared/driver'
```

Comment:
same here, the partial isnt bringing us any value here to be honest, and abstraction for no reason is pointless to me.